internal/commands/ci.go

```diff
@@ -105,6 +96,11 @@ func (o *CIOptions) Run(cmd *cobra.Command) error {
 		return err
 	}
 
+	key, err := getPublicKey(fmt.Sprintf("%s/.ssh/id_rsa.pub", o.Config.Home))
```

Comment:
Let's add a TODO to read the key as a parameter too.

Reply:
I created a story in Jira